xml/decoders/NCE_D13SR_TC.xml

```diff
@@ -34,7 +35,8 @@
   <!-- Version 8 - added D408SR V3.6 which has different function mapping then V3.5 -->
   <!-- Version 9 - added Factory Direct Trains Trackmobile -->
   <!-- Version 10 - added BLI Hi-Rail pickup and track inspection sedan -->
-  <!-- Version 10 - added N12K0a and N12K0b plus a few comments/tooltips -->  
+  <!-- Version 10 - added N12K0a and N12K0b plus a few comments/tooltips -->
+  <!-- Version 12 - high Ver ID 38, BLI Plymouth switcher -->  
```

Comment:
Please disregard this update. Problems with editor and with JMRI 4.7.3. No error in XML indicated but Recreate Index is not picking up changes for reason unknown so far.